plugins/sagemaker/README.md

```diff
@@ -1 +1,167 @@
 # Using Sagemaker with Flyte - Sagemaker plugin
+
+## Prerequisites
+Before following this example, make sure that 
+- SageMaker plugins are [enabled in flytepropeller's config](https://github.com/lyft/flytepropeller/blob/f9819ab2f4ff817ce5f8b8bb55a837cf0aeaf229/config.yaml#L35-L36)
+- You have your [AWS role set up correctly for SageMaker](https://docs.aws.amazon.com/sagemaker/latest/dg/sagemaker-roles.html#sagemaker-roles-amazonsagemakerfullaccess-policy)
+- [AWS SageMaker k8s operator](https://github.com/aws/amazon-sagemaker-operator-for-k8s) is installed in your k8s cluster
+
+
+## Define a Training Job Task of SageMaker's built-in algorithm
```

Comment:
lets provide a high level view of how this works. For example, that users can use builtin algorithms without needing to provide an image and train a model. Once you have a training job you an easily hpo it by composing a higher level hpo task from the training job.

Reply:
I also think HPO and training should be different readmes, and different examples.

So this is how I would structure
1. Train a Builtin Algorithm
2. Train a Custom Algorithm
3. HPO your training algorithm